src/components/atoms/Rating/Rating.module.scss

```
@@ -0,0 +1,14 @@
+@use '/src/styles/vars';
+@use '/src/styles/placeholders';
+
+.container {
+    display: flex;
+    @extend %flex-center;
+
+    border-radius: 20%;
+    width: 50px;
```

For repeating values in scss, assign a variable so it's quicker to modify it later. In this case, since it's a square component, we'd always change the width/height at the same time.

```suggestion
$size: 50px;
width: $size;
height: $size;
```